frameworks/mxnet/get_started_mnist_deploy.ipynb

```diff
@@ -0,0 +1,608 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Deploy a Trained Model\n",
+    "\n",
+    "In this notebook, we walk through the process of deploying a trained model to a SageMaker endpoint. If you recently ran [the notebook for training](get_started_mnist_deploy.ipynb) with %store% magic, the `model_data` can be restored. Otherwise, we retrieve the \n",
+    "model artifact from a public S3 bucket."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# setups\n",
+    "\n",
+    "import os\n",
+    "import json\n",
+    "\n",
+    "\n",
+    "import boto3\n",
+    "import sagemaker\n",
+    "from sagemaker.mxnet import MXNetModel\n",
+    "from sagemaker import get_execution_role, Session\n",
+    "\n",
+    "sess = Session()\n",
+    "role = get_execution_role()\n",
+    "\n",
+    "%store -r mx_mnist_model_data\n",
+    "\n",
+    "try: \n",
+    "    mx_mnist_model_data\n",
+    "except NameError:\n",
+    "    import json\n",
+    "    # copy a pretrained model from a public public to your default bucket\n",
+    "    with open('code/config.json', 'r') as f:\n",
+    "        CONFIG = json.load(f)\n",
+    "    \n",
+    "    bucket = CONFIG['public_bucket']\n",
+    "    s3 = boto3.client('s3')\n",
+    "    key = 'datasets/image/MNIST/model/mxnet-training-2020-11-21-01-38-01-009/model.tar.gz'\n",
+    "    target = os.path.join('/tmp', 'model.tar.gz')\n",
+    "\n",
+    "    s3.download_file(bucket, key, target)\n",
+    "    \n",
+    "    # upload to default bucket\n",
+    "    mx_mnist_model_data = sess.upload_data(\n",
+    "        path=os.path.join('/tmp','model.tar.gz'),\n",
+    "        bucket=sess.default_bucket(),\n",
+    "        key_prefix='model/mxnet')\n",
+    "\n",
+    "    \n",
+    "    "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(mx_mnist_model_data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## MXNet Model Object\n",
+    "\n",
+    "The `MXNetModel` class allows you to define an environment for making inference using your\n",
+    "model artifact. Like `MXNet` class we discussed \n",
+    "[in this notebook for training an MXNet model](\n",
+    "get_started_mnist_train.ipynb), it is high level API used to set up a docker image for your model hosting service.\n",
+    "\n",
+    "Once it is properly configured, it can be used to create a SageMaker\n",
+    "Endpoint on an EC2 instance. The SageMaker endpoint is a containerized environment that uses your trained model \n",
+    "to make inference on incoming data via RESTful API calls. \n",
+    "\n",
+    "Some common parameters used to initiate the `MXNetModel` class are:\n",
+    "- entry_point: A user defined python file to be used by the inference image as handlers of incoming requests\n",
```

Comment:
inference image is confusing here as it could the image you pass to get a prediction... but in this case I think you mean the MXNet Deep Learning Container.

Reply:
That's true. Fixed